packages/playground/assets/css/playground.css

```diff
@@ -36,9 +34,12 @@
 	transform: translateY(-100%);
 }
 
+#wp-playground-toolbar > p {
+	margin: 0;
+}
+
 #wp-playground-toolbar > a {
-	text-transform: capitalize;
-	padding: 0 0.5rem;
+	color: #1a5687;
```

Comment:
```suggestion
```

Setting a custom color breaks WordPress theme support.

![image](https://github.com/WordPress/playground-tools/assets/1199991/54ccadf3-cc66-417d-99a5-61a4944ce300)

Reply:
Interesting. Does this mean that the color contrast in WP is insufficient?